test/data/executor.start.yaml

```diff
@@ -1,3 +1,5 @@
+annotations:
+    beta.screwdriver.cd/executor: k8s
```

Comment:
> Are we supposed to have some rules about the key of the annotations? 

Depends on the work in #595, specifically the "Consume annotations to choose executor resource"

> We just assume the executor-router? knows how to parse it? This is just an example or we are sure we will use this format in executor-router?

My counter question would be: how does anyone know what value goes [in the config](https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml#L89)? The annotation value would have to match the module's name, like how we're currently doing it [in bin/server](https://github.com/screwdriver-cd/screwdriver/blob/master/bin/server#L37)